scripts/templates/variables.tf.j2

```diff
@@ -64,7 +64,20 @@ variable "{{ id }}_disabled_{{ severity }}" {
   type        = bool
   default     = null
 }
-
+{% if rule.lasting_duration is string %}
+variable "{{ id }}_lasting_duration_{{ severity }}" {
+  description = "Minimum duration that conditions must be true before raising alert (in seconds)"
```

Comment:
I don't think it is in seconds

Reply:
It's a copy/paste from https://github.com/claranet/terraform-signalfx-detectors/blob/master/modules/integration_aws-elb/variables.tf#L234

But since we put the metric in the value (eg 900s) I can remove the end of this line.